ui_framework/components/table/kui_table_cell_liner.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import { KuiTd } from './kui_table';
+
+export function KuiTableCellLiner({ children }) {
+  return <KuiTd><div className="kuiTableRowCell__liner">{children}</div></KuiTd>;
+}
+KuiTableCellLiner.propTypes = {
+  children: React.PropTypes.any
```

Comment:
I think children is either a node or an array of nodes. So I guess it would be:
```
  children: React.PropTypes.oneOf([
    React.PropTypes.arrayOf(React.PropTypes.node),
    React.PropTypes.node])
```

Too bad there isn't a built in type for children.  Thats a lot of text for such a common one.